eZ/Bundle/EzPublishCoreBundle/SiteAccess/SiteAccessMatcherRegistry.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+namespace eZ\Bundle\EzPublishCoreBundle\SiteAccess;
```

Comment:
Nitpick: Missing new line

```suggestion

namespace eZ\Bundle\EzPublishCoreBundle\SiteAccess;
```